docs/template-bugReport.md

```diff
@@ -0,0 +1,15 @@
+**Bug Report**
+
+
+**Environment**
+**Type (dev/staging/live):**
+**Version:**
```

Comment:
"Version" is only relevant when live server is being involved.
When we are talking about dev server, the closest relevant information will be the latest commit hash on `master`, i.e. something like "This bug occurs on `master` branch at commit fd27e4e".